### PR TITLE
add: `amfora-bin`

### PR DESCRIPTION
**Pull request by Henryws**
- [x]  Edit packagelist
- [x]  Add initial pacscript
- [x] Contact devs: ~~https://github.com/makeworld-the-better-one/amfora/issues/291~~
- [x] Add maintainer to pacscript: resolved to @Henryws, https://github.com/makeworld-the-better-one/amfora/issues/291#issuecomment-1005295519

### CODE REVIEW

**Comment by wizard-28** [comment]
Mark it as ready, it's marked as a draft.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/amfora-bin/amfora-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm644 amfora.png "${pkgdir}/usr/share/pixmaps/amfora.png"
+  sudo install -Dm644 amfora.desktop "${pkgdir}/usr/share/applications/amfora.desktop"
+  sudo install -Dm755 amfora_"${version}"_linux_64-bit "${pkgdir}/usr/bin/${pkgname}"
```
Comment (echometerain) [inline packages/amfora-bin/amfora-bin.pacscript:24]:
```suggestion
  mv "amfora_"${version}"_linux_64-bit" "amfora"
  sudo install -Dm755 "amfora" "${pkgdir}/usr/bin/${pkgname}"
```

**Review by echometerain — CHANGES_REQUESTED** [review]

File: packages/amfora-bin/amfora-bin.pacscript (hunk 1 above)
Comment (Henryws) [reply packages/amfora-bin/amfora-bin.pacscript:24]: Any real reason to do this?

Comment (echometerain) [reply packages/amfora-bin/amfora-bin.pacscript:24]: image
the .desktop calls `amfora`, not `amfora_"${version}"_linux_64-bit`

Comment (echometerain) [reply packages/amfora-bin/amfora-bin.pacscript:24]: also it'll be easier to call with terminal

Comment (Henryws) [reply packages/amfora-bin/amfora-bin.pacscript:24]: Yes but we are using the `install` command to install the long binary to the name `amfora`

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Verify hashes of everything you downloaded

File: packages/amfora-bin/amfora-bin.pacscript (hunk 1 above)
Comment (echometerain) [reply packages/amfora-bin/amfora-bin.pacscript:24]: oh wait I didn't see pkgname

**Review by echometerain — APPROVED** [review]

File: packages/amfora-bin/amfora-bin.pacscript [hunk 2]
```diff
@@ -11,7 +11,16 @@ maintainer="Henryws <hwengerstickel@pm.me>"
...
+  if [[ "$(sha256sum amfora.png | cut -d' ' -f 1)" != "3d029d05cff8c42e82685ce7a61fcaa2118e4cbb6a547816a7d5150868a11092" ]]; then
+    fancy_message error "Checksum for amfora.png failed"
+    return 1
+  fi
   wget -q https://raw.githubusercontent.com/makeworld-the-better-one/amfora/master/amfora.desktop
+  if [[ "$(sha256sum amfora.desktop | cut -d' ' -f 1)" != "812e1faad6f6d4817eac60d36813472afebe2980cd2e661151a3d98669274207" ]]; then
+    fancy_message error "Checksum for amfora.desktop failed"
+    return 1
+  fi
+  sed -i '/^Terminal/d' amfora.desktop
```
Comment (wizard-28) [inline packages/amfora-bin/amfora-bin.pacscript:23]: What's this for?

Comment (Henryws) [reply packages/amfora-bin/amfora-bin.pacscript:23]: To remove the terminal line discussed in Discord

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**